central/alert/datastore/datastore_impl.go

```diff
@@ -96,6 +96,7 @@ func (ds *datastoreImpl) ListAlerts(ctx context.Context, request *v1.ListAlertsR
 	}
 
 	paginated.FillPagination(q, request.GetPagination(), math.MaxInt32)
```

Comment:
Maybe in the future we combine `FillDefaultSortOption` and `FillPagination`, with the default sort option as an optional parameter, because the sort option is part of the pagination struct.